server/src/modules/Question/question.controller.ts

```diff
@@ -42,4 +43,9 @@ export class QuestionController {
     async delete(@Param('id') id: string) {
         return await this.questionService.delete(id);
     }
+
+    @Post('reply/:askedQuestionId')
+    reply(@Body() answer: string, @Param() { askedQuestionId }): Promise<AnswerDto> {
```

Comment:
Do we need this endpoint? I thought we communicate with clients through WebSockets.
Probably ask/reply methods in service are enough. Please check this with the team who developed AskedQuestions.

Reply:
Seems reasonable, I'll remove that endpoint and let them decide how to use service on their own